SPECS/qemu-kvm/qemu-kvm.spec

```diff
@@ -161,6 +163,9 @@ chmod 755 %{buildroot}%{_bindir}/qemu
 %{_bindir}/qemu-nbd
 
 %changelog
+* Fri Mar 23 2021 Neha Agarwal <nehaagarwal@microsoft.com> - 4.2.0-27
```

Comment:
No weekend for us yet, it's still Tuesday.

Reply:
Let people dream, Henry.:)